client/src/features/app/googleMaps.js

```diff
@@ -0,0 +1,89 @@
+var googleMapsAPIKey = process.env.REACT_APP_GOOGLE_MAPS_API_KEY;
+if (!googleMapsAPIKey) throw new Error("googleMapsAPIKey environment variable required");
+
+function loadScript() {
+  var script = document.createElement('script');
+  script.type = 'text/javascript';
+  script.src = 'https://maps.googleapis.com/maps/api/js?key='+googleMapsAPIKey+'&libraries=places&callback=googleMapsLoaded'; //& needed
+  document.body.appendChild(script);
+}
+
+window.googleMapsLoaded = () => {
+  initMap();
+}
+
+var map;
```

Comment:
It might be best to avoid var and use const/let

Reply:
I've used var a lot in this code, but changing to lets still work, since we don't want the values fixed?